src/main/docker/gitlab/README.md

```diff
@@ -0,0 +1,48 @@
+# Local GitLab and GitLab CI setup
+
+First, configure the hostname and external URL in the `gitlab-gitlabci.yml` file and run the following command to start GitLab and a GitLab Runner in a Docker container:
+```bash
+docker-compose -f src/main/docker/gitlab-gitlabci.yml up --build -d
+```
+
+Then log on to https://localhost/ with the password (`sudo docker exec -it NAME_OF_THE_CONTAINER grep 'Password:' /etc/gitlab/initial_root_password`) and go to https://localhost/admin/runners.
+Click on Register an instance runner and copy the registration token.
+Open a shell into the container:
+`````bash
+docker ps
+docker exec -it NAME_OF_THE_CONTAINER /bin/bash
```

Comment:
```suggestion
docker exec -it gitlab-runner /bin/bash
```
This only works, if you adapt the docker-compose file according to my suggestions.